sdk/test/UnitTests/Custom/Runtime/BearerTokenSignerTests.cs

```diff
@@ -230,49 +230,16 @@ public async Task SignerShouldOverrideExistingHeaderAsync()
         [TestCategory("Runtime")]
         public void SignerRequiresAToken()
         {
-            var mockConfig = new MockClientConfig
-            {
-                AWSTokenProvider = new AWSTokenProviderChain() // chain is empty
-            };
-
-            var pipeline = CreateMockPipeline();
-            var context = CreateTestContext(new BearerTokenSigner(), null, mockConfig);
-
-            Exception exception = null;
-
-            try
-            {
-                pipeline.InvokeSync(context);
-            }
-            catch (Exception e)
-            {
-                exception = e;
-            }
-
-            Assert.IsNotNull(exception);
-            Assert.IsInstanceOfType(exception, typeof(AmazonClientException));
-            Assert.AreEqual(exception.Message, "No Token found. Operation requires a Bearer token.");
-        }
-
-#if AWS_ASYNC_API
-        [TestMethod]
-        [TestCategory("UnitTest")]
-        [TestCategory("Runtime")]
-        public async Task SignerRequiresATokenAsync()
-        {
-            var mockConfig = new MockClientConfig
-            {
-                AWSTokenProvider = new AWSTokenProviderChain() // chain is empty
-            };
-
-            var pipeline = CreateMockPipeline();
-            var context = CreateTestContext(new BearerTokenSigner(), null, mockConfig);
-
+            var signer = new BearerTokenSigner();
```

Comment:
This test validates the signer throws an error when the access token is invalid, but it was invoking the request pipeline to achieve that.

I refactored it to call the signer class directly.